en/components/toc.yml

```diff
@@ -198,6 +198,9 @@
     - name: Conditional Cell Styling
       href: hierarchicalGrid/conditional_cell_styling.md
       new: true
+    - name: Load on Demand
+      href: hierarchicalGrid/hierarchical_grid_load_on_demand.md
```

Comment:
Currently all feature topics for the respective grids contain only the name of the feature (the respective grid is implied by the folder name), so I believe we can rename the md file
from **hierarchical_grid_load_on_demand.md**
to **load_on_demand.md**.